howto/setup-secret-store/json-secret-store.md

```diff
@@ -0,0 +1,104 @@
+# Secret Store for JSON (Development)
+
+This document shows how to enable JSON secret store using [Dapr Secrets Component](../../concepts/secrets/README.md) for Development scenarios in Standalone mode. This Dapr secret store component reads plain text JSON from a given file and does not use authentication.
+
+## Contents
+
+- [Create JSON file to hold the secrets](#create-json-file-to-hold-the-secrets)
+- [Use JSON secret store in Standalone mode](#use-json-secret-store-in-standalone-mode)
+- [References](#references)
+
+## Create JSON file to hold the secrets
+
+This creates new JSON file to hold the secrets.
+
+1. Create a json file (i.e. secrets.json) with the following contents:
+
+```json
+{
+    "redisPassword": "your redis passphrase"
+}
+```
+
+## Use JSON secret store in Standalone mode
+
+This section walks you through how to enable an JSON secret store to store a password to access a Redis state store in Standalone mode.
+
+1. Create a components directory in your application root
```

Comment:
Can skip this stage and in the next one just say to add the yaml to "your" components directory (https://github.com/dapr/cli/issues/340 will be changing the behavior of components directory creation)